packages/parsley/parsley.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+]
+homepage: "https://github.com/ghilesZ/parsley"
+bug-reports: "https://github.com/ghilesZ/parsley/issues"
+dev-repo: "git+https://github.com/ghilesZ/parsley"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.05.0"}
```
It seems like this package requires at least OCaml 4.05:
```
#=== ERROR while compiling parsley.0.0.1 ======================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/ghilesZ/parsley/archive/0.0.1.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/parsley.0.0.1
# command              ~/.opam/4.04/bin/dune build -p parsley -j 2
# exit-code            1
# env-file             ~/.opam/log/parsley-23-7d4558.env
# output-file          ~/.opam/log/parsley-23-7d4558.out
### output ###
#       ocamlc lib/.parsley.objs/byte/parsley.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.parsley.objs/byte -I /home/opam/.opam/4.04/lib/zarith -no-alias-deps -open Parsley__ -o lib/.parsley.objs/byte/parsley.cmo -c -impl lib/parsley.ml)
# File "lib/parsley.ml", line 180, characters 38-55:
# Error: Unbound value int_of_string_opt
```

Reply:
Ok i have corrected it, (in fact it requires at least OCaml 4.08, as it uses the Result module)